components/global-styles.scss

```diff
@@ -1,5 +1,5 @@
 body {
   background: #eee;
   font-family: 'Space Mono', monospace;
-  padding: 3vw;
-}
+  padding: 1vw 3vw;
```

Comment:
`vw` betyr viewport width, og er en prosent av bredden på viewporten din. Her bruker du den for å spesifisere høyden på paddingen din.

`vw` har et søsken, `vh` - viewport height - som kanskje er mer relevant

Eller for å være ærlig - viewport units er ikke alltid den beste måleenheten for padding på denne måten. Er det en prosent av størrelsen av skjermen som skal spesifisere bredden på paddingen din liksom? Vurder heller å bruke em eller piksler.

Hvis du vil da. Rent teknisk er dette helt innafor.